sm/webapp/static/js/chart.js

```diff
@@ -1,112 +1,58 @@
-        }
-        for (var i = 0; i < data["sample"]["inds"].length; i++) {
-            var grid_ind = data["sample"]["inds"][i];
-            centr_int_series[grid_ind] = data["sample"]["ints"][i]
-        }
-
-        // prepare chart data structure
-        var chartData = [];
-        for (var i = 0; i < points_n; i++) {
-            chartData.push({"peak_mz": grid[i],
-                            "prof_peak_int": prof_int_series[i],
-                            "centr_peak_int": centr_int_series[i]});
-        }
-
-        // create guides from centroids
-        var guides = data["sample"]["inds"].map(function(grid_ind, ind, arr) {
-            return {"category" : grid[grid_ind],
-                    "boldLabel" : "True",
-                    "color" : "black",
-                    "dashLength" : 5,
-                    "lineAlpha" : 1,
-                    "label" : grid[grid_ind]}
-        });
-
-        AmCharts.makeChart("peaks-line-chart", {
-            type: "serial",
-            pathToImages: "http://sm-engine-webapp.s3-website-eu-west-1.amazonaws.com/js/amcharts/images/",
-            dataProvider: chartData,
-            categoryField: "peak_mz",
-
-            categoryAxis: {
-                dashLength: 1,
-                labelsEnabled: false,
-                fontSize: 14,
-                guides: guides,
-            },
-
-            valueAxes: [{
-                id: "val_axis",
-                title: "Intensity (a.u.)",
-                fontSize: 14,
-                maximum: 120,
-                axisThickness: 1.5,
-                dashLength: 5,
-                gridCount: 10,
-                dashLength: 1,
-            }],
-
-            graphs: [{
-                id: "theor_int",
-                valueField: "prof_peak_int",
-                title: "Theoretical",
-                lineColor: "blue",
-                type: "smoothedLine",
-
-            },{
-                id: "sample_int",
-                valueField: "centr_peak_int",
-                title: "Sample",
-                lineColor: "red",
-                type: "column",
-                fixedColumnWidth: 5,
-                fillAlphas: 1,
-            }],
-
-            chartScrollbar : {
-                //updateOnReleaseOnly: true,
-            },
-
-            chartCursor: {
-                cursorPosition: "mouse",
-            },
-
-            legend: {
-              fontSize: 14,
-              markerSize: 15,
-              useGraphSettings: true,
-              position: "bottom",
-              align: "center",
+        var layout = {
+            xaxis: {'range': [min_mz, max_mz]},
+            yaxis: {title: 'Intensity (a. u.)', rangemode: 'nonnegative'},
+            legend: {x: 0.5, y: -0.2, xanchor: 'center', yanchor: 'top',
+                     orientation: 'h', traceorder: 'reversed'},
+            margin: {t: 20, b: 20},
+            font: {size: 16}
+        };
+
+        var sampleData = centroidPlot(data['sample'])
```

Comment:
Not yet, I'd rather avoid discussions like https://www.reddit.com/r/programming/comments/sag8p/crockford_on_bootstraps_semicolon_omission/